frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts

```diff
@@ -127,61 +129,106 @@ export const TEMPLATE = {
     metadataName: 'rhel6-server-small',
     os: 'Red Hat Enterprise Linux 6.0 or higher',
     supportLevel: 'Full',
+    exampleImgUrl: urls.RHEL_IMAGE_LINK,
```

Comment:
`urls.RHEL_IMAGE_LINK` does not exist yet, something is missing.

Reply:
I see it added 11/2/21 by Kenigsber in revision e289cd6582335ad2c6e24faced6e81cf03a2b8ef